localstack/services/awslambda/lambda_api.py

```diff
@@ -803,10 +805,10 @@ def forward_to_fallback_url(func_arn, data):
         }
         aws_stack.create_dynamodb_table(table_name, partition_key='id')
         dynamodb.put_item(TableName=table_name, Item=item)
-        return ''
+        return '', lambda_name
     if re.match(r'^https?://.+', config.LAMBDA_FALLBACK_URL):
         response = safe_requests.post(config.LAMBDA_FALLBACK_URL, data)
```

Comment:
I think the idea of #1971 is that we send the function name along with the request that is being sent to `config.LAMBDA_FALLBACK_URL`. Something like:
```
headers = {'lambda-function-name': lambda_name}
response = safe_requests.post(config.LAMBDA_FALLBACK_URL, data, headers=headers)
```

Same for DynamoDB, see comment above.

So, I guess we can probably also remove the additional `, lambda_name` result returned from this function.